### PR TITLE
Minor code improvements

### LINKED ISSUE #109: Minor Code Improvements
**Involved artifacts**
• x/feeabs/ante/decorate.go
• x/feeabs/ibc.go 

**Description**
During the code review, several minor code improvements were identified in the Fee Abstraction module. These improvements aim to enhance code readability, maintainability, and adherence to best practices.

### PR DESCRIPTION
**Pull request by minhngoc274**
Close #109

### CODE REVIEW

File: x/feeabs/keeper/epoch.go [hunk 1]
```diff
@@ -64,12 +64,6 @@ func (k Keeper) setEpochInfo(ctx sdk.Context, epoch types.EpochInfo) {
 	store.Set(append(types.KeyPrefixEpoch, []byte(epoch.Identifier)...), value)
 }
 
-// DeleteEpochInfo delete epoch info.
-func (k Keeper) DeleteEpochInfo(ctx sdk.Context, identifier string) {
```
Comment (nghuyenthevinh2000) [inline x/feeabs/keeper/epoch.go:68]: Does removing this function affect logic?

File: x/feeabs/keeper/epoch.go [hunk 2]
```diff
@@ -104,18 +98,6 @@ func (k Keeper) AllEpochInfos(ctx sdk.Context) []types.EpochInfo {
 	return epochs
 }
 
-// NumBlocksSinceEpochStart returns the number of blocks since the epoch started.
-// if the epoch started on block N, then calling this during block N (after BeforeEpochStart)
-// would return 0.
-// Calling it any point in block N+1 (assuming the epoch doesn't increment) would return 1.
-func (k Keeper) NumBlocksSinceEpochStart(ctx sdk.Context, identifier string) (int64, error) {
```
Comment (nghuyenthevinh2000) [inline x/feeabs/keeper/epoch.go:111]: Does removing this function affect logic?

File: x/feeabs/keeper/epoch.go (hunk 1 above)
Comment (minhngoc274) [reply x/feeabs/keeper/epoch.go:68]: The function is unused and can be safely removed

File: x/feeabs/keeper/epoch.go (hunk 2 above)
Comment (minhngoc274) [reply x/feeabs/keeper/epoch.go:111]: Same with the previous comment

**Review by nghuyenthevinh2000 — APPROVED** [review]

**Pull request merged**